pkg/logging/slog.go

```diff
@@ -95,6 +100,11 @@ func replaceLevel(groups []string, a slog.Attr) slog.Attr {
 			Key:   a.Key,
 			Value: slog.StringValue(strings.ToLower(a.Value.String())),
 		}
+	case logrErrorKey:
```

Comment:
the name of this function (`replaceLevel`) is no longer correct (probably already before this change).

Reply:
Yeah, makes sense. I've generalized the two function names, and made one build on top of the other to ensure consistency and reduce code duplication.